test/docs/dash-test-framework.md

```diff
@@ -0,0 +1,54 @@
+
+# **DASH Testing Framework**
```

Comment:
This gives the impression that there is "one framework" when actually there are at least three:
1. HW-based, i.e. Mircea's testbed
2. PTF
3. SAI-Challenger
I think it's incumbent on us to document all this in proper context. I recommend we hold off trying to "document the frameworks" until we're prepared to refresh and level the entire topic.

Reply:
Now that SAI-Challenger PR is merged, we can discuss the best way to include documentation for it.